README.md

```diff
@@ -66,6 +66,14 @@ source root.
 Remove the `#!/usr/bin/perl`, `use strict;` or `use warnings;` from
 the beginning of your example before inserting them into the POD.
 
+If ["match\_barrier"](#match_barrier) is also set, it instead removes all lines up-to
+and including the line matched by ["match\_barrier"](#match_barrier).
+
+## match\_barrier
```

Comment:
This does not match the POD, I think just need to re-run `dzil build` to re-generate.

Reply:
Fixed.  Also found a similar typo in the tests which rendered it useless.  Not sure how things got out of synch